docs/Project-Configuration/Project-Commands.md

```diff
@@ -2,6 +2,26 @@
 
 Project Commands are the fundamental units of functionality in an Evergreen task.
 
+# Basic Command Structure
+
+```yaml
+- command: shell.exec
+  display_name: run my cool script ## optional 
+  type: system ## optional
+  timeout_secs: 10 ## optional
+  params:
+    script: echo "my script"
+```
+Explanation:
+
+-   `command`: a command name predefined by evergreen
```

Comment:
Predefined might be a little vague in that it's unclear what they can choose from? Maybe clarify that they can pick from the list of commands documented below.